docs/standard/changelog.rst

```diff
@@ -6,25 +6,66 @@ ChangeLog
 
 .. include:: warningbox.rst
 
-[Unreleased]
-============
+0.4 - 2024-05-04
+================ 
+
+Added
+-----
+* Separation of metadata and data into statements and records
+* ``declaration`` and ``declarationSubject`` to model declarations
+* ``recordId`` and ``recordStatus`` to model change over time 
+* New documentation pages: 
+
+  * Dates guidance 
+  * Generating statements
+  * Record identifiers
+  * Representing trusts 
+  * Representing nominees 
+  * Representing record updates
 
 Changed
 -------
-- Removed ``securitiesListing`` from required fields in ``hasPublicListing``.
-- Scenario added to 'Representing state-owned enterprises' page covering ownership or control by a regional government.
+* valid date formats for all date fields restricted 
+* Schema and codelist titles and descriptions updated
+* Documentation images and text updated 
+* Documentation guidance restructured into modelling requirements and system requirements
+* Ownership-or-control statements replaced with Relationship statements
+* Statement fields:
```

Comment:
```suggestion
* Statement fields:

```
I think you need to include a blank line under this to make the sublist appear correctly in the built page.